setup.py

```diff
@@ -1,5 +1,22 @@
+import sys
 from setuptools import setup, find_packages
 
+
+def get_requirements():
+    requirements = [
+        'andi>=0.3',
+        'attrs',
+        'parsel',
+        'tenacity',
+        'web-poet',
+    ]
+
+    if not sys.version_info >= (3, 7):
+        requirements.append('dataclasses')
```

Comment:
I think that's better to use https://www.python.org/dev/peps/pep-0508/#environment-markers; dynamic code in setup.py may not be the best for wheels - I'm not sure it would cause an actual issue here though.

Reply:
#TIL I'll give it a try

Reply:
Done.